certify-sunbird-insurance.properties

```diff
@@ -9,108 +9,47 @@ mosip.certify.integration.audit-plugin=LoggerAuditService
 
 ##Life Insurance Credential Configurations
 
 mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:api.dev1.mosip.net:identity-service:8ebda1d0-665b-4bb7-abc7-d4bf56b6ee09
-mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/life-insurance-credential.vm
-mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=${mosip.sunbird.url}/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=${spring_config_url_env}/${spring.application.name}/${active_profile_env}/${spring_config_label_env}/life-insurance-credential.vm
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/
 mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-id=did:schema:e1f8df1d-c46c-483c-a882-3fc823666b9f
 mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-version=1.0.0
-mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=${mosip.sunbird.url}/api/v1/Insurance/search
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/search
 
 ##Insurance Credential Configurations
 mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.static-value-map.issuerId=did:web:api.dev1.mosip.net:identity-service:8ebda1d0-665b-4bb7-abc7-d4bf56b6ee09
-mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/insurance-credential.vm
-mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-get-url=${mosip.sunbird.url}/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.template-url=${spring_config_url_env}/${spring.application.name}/${active_profile_env}/${spring_config_label_env}/insurance-credential.vm
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-get-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/
 mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-id=did:schema:e1f8df1d-c46c-483c-a882-3fc823666b9f
 mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.cred-schema-version=1.0.0
-mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=${mosip.sunbird.url}/api/v1/Insurance/search
+mosip.certify.vciplugin.sunbird-rc.credential-type.InsuranceCredential.registry-search-url=http://demo-registry.sunbird-registry:8081/api/v1/Insurance/search
```

Comment:
Do we need the above domain name change ? Can we not take it from configmap like usual ?

Reply:
Reverted all changes to take the domain names from configmap